### DISCUSSION TITLE
RichtTextEdit without Editor?

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#3906] [Q&A]
Hello,
I would like to have the RichtTextEdit component to have no editor, but changing the markup like this

```
<RichTextEdit @ref="richTextEdit" ContentChanged="OnFileContentChanged">
                                        <Toolbar></Toolbar>
                                        <Editor>
                                            @((MarkupString)String.Join("<br />", FileContentRecognized))
                                        </Editor>
                                    </RichTextEdit>
```

displays an empty toolbar. Is there a way to have no toolbar?

**Comment by stsrki** [comment]
`<Toolbar></Toolbar>` is used to override toolbar buttons. So even if you don't define anything inside of it, the Toolbar render fragment is still non-null so it will be used.

Maybe you can hide it with CSS?

```
.ql-toolbar {
  display: none;
}
```
